Cargo.toml

```diff
@@ -26,13 +26,13 @@ pyo3 = "0.23.3"
 serde_json = "1.0.111"
 cargo_toml = "0.20.4"
 thiserror = "2.0.6"
-hugr = "0.14.1"
-hugr-cli = "0.14.1"
+hugr = "0.14.4"
+hugr-cli = "0.14.4"
 inkwell = "0.5.0"
 
 [patch.crates-io]
 
 # Uncomment these to test the latest dependency version during development
- hugr = { git = "https://github.com/CQCL/hugr", rev = "5e7c81e" }
- hugr-cli = { git = "https://github.com/CQCL/hugr", rev = "5e7c81e" }
- hugr-llvm = { git = "https://github.com/CQCL/hugr", rev = "5e7c81e" }
+ hugr = { git = "https://github.com/CQCL/hugr", rev = "4e6b6d8" }
+ hugr-cli = { git = "https://github.com/CQCL/hugr", rev = "4e6b6d8" }
+ hugr-llvm = { git = "https://github.com/CQCL/hugr", rev = "4e6b6d8" }
```

Comment:
Should we wait to merge this until those are released?

Reply:
I want to wait until hugr-py release but not too fussed about validation dependencies